great_expectations/core/expectation_validation_result.py

```diff
@@ -514,6 +514,7 @@ def __init__(  # noqa: PLR0913
         statistics: Optional[dict] = None,
         meta: Optional[ExpectationSuiteValidationResult | dict] = None,
         batch_id: Optional[str] = None,
+        result_url: Optional[str] = None,
```

Comment:
I don't love expanding the definition of what an EVR is but this seems like the path of least resistance to pass the result URL down the call stack